semgrep/tests/e2e/test_cli_test.py

```diff
@@ -35,3 +35,16 @@ def test_cli_test_suffixes(run_semgrep_in_tmp, snapshot):
         results,
         "results.json",
     )
+
+
+def test_cli_test_multiline_annotations(run_semgrep_in_tmp, snapshot):
+    results = run_semgrep_in_tmp(
+        "rules/cli_test/multiple_annotations/",
+        options=["--test"],
+        target_name="cli_test/multiple_annotations/",
+        output_format="text",
+    )
+    snapshot.assert_match(
+        results,
+        "results.json",
```

Comment:
This isn't actually outputting JSON - I would either change `output_format="json"` or use `results.txt` :+1: